go-controller/pkg/testing/testing.go

```diff
@@ -16,6 +17,10 @@ type FakeExec struct {
 
 // NewFakeExec returns a new FakeExec with a default LookPathFunc
 func NewFakeExec() *FakeExec {
+	// Gomega's default string diff behavior makes it impossible to figure
+	// out what fake command is failing, so turn it off
+	format.TruncatedDiff = false
```

Comment:
Yay, but maybe move this to an `init` func? AFAIK it applies globally and doesn't get reset between tests so it would be confusing to have it only take effect after the first `FakeExec` creation.